README.md

```diff
@@ -3,68 +3,87 @@
 [![Build Status](https://travis-ci.org/cloudinary/cloudinary-vue.svg?branch=master)](https://travis-ci.org/cloudinary/cloudinary-vue)
 
 Cloudinary is a [cloud-based service](https://cloudinary.com/solutions) that provides an end-to-end image and video management solution
-including uploads,storage, manipulations, optimizations and delivery. 
```

Comment:
These extra spaces were line breaks, are we sure we want to remove them? it makes for some long lines in the read me.

Reply:
linting problem :(

Reply:
Up to you if you want to fix it, it's not critical.